src/package.c

```diff
@@ -975,6 +975,64 @@ int apk_ipkg_add_script(struct apk_installed_package *ipkg,
 	return 0;
 }
 
+static void apk_run_script(const char *script, int dirfd)
+
+{
+	int status;
+	pid_t pid;
+	char *script_args[] = { "/bin/sh", "-c", "", NULL };
+	static char * const environment[] = {
+		"PATH=/usr/sbin:/usr/bin:/sbin:/bin",
+		NULL
+	};
+	script_args[2] = (char *) script;
```

Comment:
Much of things here are copied from apk_ipkg_add_script(). Please make this new function general purpose enough, and convert apk_ipkg_run_script() use this so we don't duplicate code.